src/Commands/Microsoft365Groups/NewMicrosoft365Group.cs

```diff
@@ -115,8 +133,16 @@ protected override void ExecuteCmdlet()
                     GroupTypes = new string[] { "Unified" },
                     PreferredDataLocation = PreferredDataLocation,
                     PreferredLanguage = PreferredLanguage,
+                    MembershipRule = DynamicMembershipRule
```

Comment:
What happens if the `DynamicMembershipRule` is empty here ? Does it work as it works currently ?

Reply:
Correct. At first I also had a SwitchParameter to indicate it needed to be a dynamic membership, but since the DynamicMembershipRule is required, I figured that could just as well already function as a trigger to indicate it needs to be dynamic instead of assigned.